.jenkins/pytorch/test.sh

```diff
@@ -586,6 +586,8 @@ test_dynamo() {
 }
 
 test_functorch() {
+  # functorch testing dependency
+  pip install networkx
```

Comment:
To avoid flaky network failure, can you please instead add it to https://github.com/pytorch/pytorch/blob/master/.circleci/docker/requirements-ci.txt

This way it will be baked into docker image

Reply:
Yes, will do, thanks for the suggestion

Reply:
You can delete it from this file if you've added it to requirements-ci.txt, don't you?

Reply:
Good point, let me submit a follow-up